libs/jsonforms-components/src/lib/Controls/FormStepper/FormStepper.spec.tsx

```diff
@@ -0,0 +1,353 @@
+            {
+              type: 'Control',
+              scope: '#/properties/carBrands',
+              options: {
+                enumContext: {
+                  key: 'car-brands',
+                  url: 'https://parallelum.com.br/fipe/api/v1/carros/marcas',
+                  values: 'nome',
+                },
+              },
+            },
+            {
+              type: 'Control',
+              scope: '#/properties/dogBreeds',
+              options: {
+                enumContext: {
+                  key: 'dog-list',
+                  url: 'https://dog.ceo/api/breeds/list/all',
+                  location: 'message',
+                  type: 'keys',
+                },
+              },
+            },
+          ],
+        },
+        {
+          type: 'HorizontalLayout',
+          elements: [
+            {
+              type: 'Control',
+              scope: '#/properties/basketballPlayers',
+              options: {
+                autocomplete: true,
+                enumContext: {
+                  key: 'basketball-players',
+                  location: 'data',
+                  url: 'https://www.balldontlie.io/api/v1/players',
+                  values: ['first_name', 'last_name'],
+                },
+              },
+            },
+            {
+              type: 'Control',
+              scope: '#/properties/countries',
+              options: {
+                autocomplete: true,
+                enumContext: {
+                  key: 'countries',
+                },
+              },
+            },
+          ],
+        },
+      ],
+    },
+    {
+      type: 'Category',
+      i18n: 'address',
+      label: 'Address Information',
+      elements: [
+        {
+          type: 'HorizontalLayout',
+          elements: [
+            {
+              type: 'Control',
+              scope: '#/properties/address/properties/street',
+            },
+            {
+              type: 'Control',
+              scope: '#/properties/address/properties/streetNumber',
+            },
+          ],
+        },
+        {
+          type: 'HorizontalLayout',
+          elements: [
+            {
+              type: 'Control',
+              scope: '#/properties/address/properties/city',
+            },
+            {
+              type: 'Control',
+              scope: '#/properties/address/properties/postalCode',
+            },
+          ],
+        },
+      ],
+      rule: {
+        effect: 'SHOW',
+        condition: {
+          scope: '#/properties/provideAddress',
+          schema: { const: true },
+        },
+      },
+    },
+    {
+      type: 'Category',
+      label: 'Additional Information',
+      elements: [
+        {
+          type: 'Control',
+          scope: '#/properties/vegetarianOptions/properties/vegan',
+        },
+        {
+          type: 'Control',
+          scope: '#/properties/vegetarianOptions/properties/favoriteVegetable',
+        },
+      ],
+    },
+  ],
+};
```

Comment:
Again, your tests do not need such elaborate schema's.  Either make them simpler or have you tests use all of the fields.  The way you have it makes it very hard to determine what exactly is being tested and requires the reader to spend too much time figuring out that most of it is unnecessary.